### DISCUSSION TITLE
ccminer not enabled for CPU/Verus

### DISCUSSION THREAD

**Discussion by lohash** [RainbowMiner/RainbowMiner#1674] [Q&A] [answered]
I can’t get ccminerVerus CPU to enable at all. I see it in the /Miners folder, and even set it as my preferred miner, but Rainbow never benchmarks it or even tries to use it. It typically gets the highest hash rate and no dev fee!

Thanks for the help and an awesome product

**Comment by RainbowMiner** [comment]
- CcminerVerus is the Nvidia GPU only version.
- CpuminerVerus is the CPU version.

> Thanks for the help and an awesome product

Thank you :)

**Comment by lohash** [comment]
Ah ha! In that case, any way to figure out why CpuminerVerus is failing benchmark? I manually ran the same command I found in the log when it tried to benchmark it and it runs fine. Maybe there’s an issue on the monitoring end?

**Comment by RainbowMiner** [comment]
Ok, I'll double check - could you please upload a Debug file? Just click "Debug file" at http://localhost:4000 (it's on the left hand side)

**Comment by lohash** [comment, marked as answer]
I got it fixed! I’m on Ubuntu, somehow several of my /Stats/Miners files had their owner changed to root so RainbowMiner couldn’t access them. After chown‘ing them back to the user RainbowMiner runs as, they all started benchmarking including CpuminerVerus. Thanks again!

**Comment by RainbowMiner** [comment]
That's great! I am happy, that it's working now.
